addons/default/visiosoft/advs-module/src/Adv/AdvsExport.php

```diff
@@ -33,10 +33,9 @@ public function collection()
 			->leftJoin('cats_category_translations', 'cats_category.id', 'cats_category_translations.entry_id')
 			->leftJoin('location_countries_translations','advs_advs.country_id', 'location_countries_translations.entry_id')
 			->leftJoin('location_cities_translations','advs_advs.city', 'location_cities_translations.entry_id')
-			->leftJoin('location_districts_translations','advs_advs.district', 'location_districts_translations.entry_id')
 			->whereIn('cats_category_translations.locale', array(Request()->session()->get('_locale'), setting_value('streams::default_locale'), 'en'))
 			->whereIn('advs_advs_translations.locale', array(Request()->session()->get('_locale'), setting_value('streams::default_locale'), 'en'))
-			->select(['advs_advs.*', 'location_countries_translations.name as country', 'location_cities_translations.name as city_name', 'location_districts_translations.name as district', DB::raw("group_concat(default_cats_category_translations.name SEPARATOR ', ') as categories")])
+			->select(['advs_advs.*', 'location_countries_translations.name as country', 'location_cities_translations.name as city_name', DB::raw("group_concat(default_cats_category_translations.name SEPARATOR ', ') as categories")])
```

Comment:
do not use all *

Reply:
https://github.com/openclassify/openclassify/pull/781/commits/c3a1b293369aa7420bf75a3e4194e116818a1166#diff-a3d90307d6a107f598170b9200d42765b3fd379a3c3a09700a61a2366756cef2R38